### ISSUE TITLE
List should have listitem elements

### ISSUE THREAD

**Issue by Kyzyl-ool** [gravity-ui/uikit#1338]
https://dequeuniversity.com/rules/axe/4.7/aria-required-children?application=axeAPI
https://preview.gravity-ui.com/uikit/?path=/story/components-data-display-list--sortable

```
Element has children which are not allowed: [role=button]
```

tags: wcag2a, wcag131

**Comment by amje** [comment]
Cannot reproduce
